### PR TITLE
kubevirtci,periodic: prowjob to publish new CentOS stream base

### PR DESCRIPTION
**Pull request by brianmcarey**
**What this PR does / why we need it**:

This new periodic job will publish a new centos stream base image for kubevirtci every week and create a PR to update the tag in the kubevirtci repo.

This relies on https://github.com/kubevirt/kubevirtci/pull/1140 being merged into the kubevirtci repo

Also update the existing check-provision jobs to run just the kubernetes phase of provisioning.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @dhiller 

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by brianmcarey** [comment]
Successful run of the new periodic lane - https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirtci-bump-centos-base/1764615724900290560

**Review by dhiller — APPROVED** [review]
@brianmcarey looks good to me! One nit from me would be to extract a preset from the commonly repeated environment variables.

/approve

/hold in case you want to address the nit, otherwise unhold

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml [hunk 1]
```diff
@@ -270,7 +270,12 @@ presubmits:
         - /usr/local/bin/runner.sh
         - /bin/sh
         - -c
-        - cd cluster-provision/k8s/1.27 && KUBEVIRT_PSA='true' ../provision.sh
+        - cd cluster-provision/k8s/1.27 && ../provision.sh
+        env:
+        - name: KUBEVIRT_PSA
```
[the remark this answers is not in the document]
Comment (brianmcarey) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml]: Preset is a good call - added one for the common environment variables.

**Review by dhiller — APPROVED** [review]
Thanks @brianmcarey

/unhold

**Pull request merged**